sdks/java/io/iceberg/src/test/java/org/apache/beam/sdk/io/iceberg/catalog/BigQueryMetastoreCatalogIT.java

```diff
@@ -62,11 +62,12 @@ public static void deleteDataset() {
 
   @Override
   public String tableId() {
-    return DATASET + "." + testName.getMethodName() + "_" + SALT;
+    return DATASET + "." + testName.getMethodName() + "_" + salt;
   }
 
   @Override
   public Catalog createCatalog() {
```

Comment:
since createCatalog is called in setup() which is a non-static method, salt should also be non-static member

Reply:
Done